clipplexAPI.py

```
@@ -130,117 +635,488 @@ def __init__(self, media_path: str, time: str, fps: float):
 
     def _download_frames(self):
         cmd = f"ffmpeg -ss {self.time} -i {self.media_path} -vframes {self.fps} -qscale:v 2 {MEDIA_STATIC_PATH}/images/{self.time.replace(':','_')}_%03d.jpg"
-        a = subprocess.call(cmd, shell=True, stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL)
+        subprocess.call(cmd, shell=True, stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL)
```

Ruff (0.16.2)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)